bears/java/JavaPMDBear.py

```diff
@@ -18,6 +18,7 @@ class JavaPMDBear:
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Simplification', 'Unreachable Code'}
```

Comment:
Detects duplicate code.
If `empty try/catch/finally/switch statements` qualifies as `smell` then that too